docs/detections/rules-ui-create.asciidoc

```diff
@@ -383,7 +387,7 @@ NOTE: The `max_signals` default value is 100. You can modify it using the <<rule
 The {esql} rule has the following limitations:
 
 - If your {esql} query creates new fields that aren’t part of the ECS schema, they won’t be mapped to the alerts index and you can't search or filter for them from the Alerts table. As a workaround, create <<runtime-fields,runtime fields>>. 
-- If your {esql} query creates new fields that aren’t in the query’s source index, they can’t be added to the rule’s <<rule-ui-advanced-params,custom highlighted fields>>.
+- New fields created by your query won't be displayed in the Highlighted fields section of the alert details flyout. To ensure they're visible while you're investigating alerts, add the new fields to the rule's  <<rule-ui-advanced-params,custom highlighted fields>>. 
```

Comment:
I don't think we need this. This statement is applicable to any other rule and case. User won't see any field in highlighted section in alerts flyout, unless selects it in highlighted fields control.

Reply:
I thought every rule has a default set of relevant fields that display in the Highlighted fields section of the alert details flyout, and users can add to it as they like. Is that no longer the case?

Reply:
Yes, some rule have. But this section is not about it. 
It is not limitation that newly created fields are not displayed in highlighted fields section by default.
That's true for any random field for any rule type. Unless user adds them specifically in Custom highlighted fields  controller, they won't be displayed in Highlighted fields section of the alert details flyout

Reply:
Ah, ok gotcha. You're saying that this section should only convey limitations that are specific to the ES|QL rule type, and that the Highlighted fields section works the way it does for all rule types, not just the ES|QL.

Since the Highlighted fields section functions the same for all rule types (i.e., it only shows a default set of relevant fields unless you specify additional fields), I wonder if it's necessary to highlight in the ES|QL rule docs that newly-created ES|QL fields can be selected as custom highlighted fields. We're documenting this change in the 8.14 release notes, and possibly the What's new page as well. What are your thoughts, @vitaliidm

Reply:
Yes, we should highlight it. That was the main objective of the improvement: to  make these fields available to select in Custom highlighted fields component.

Reply:
@nastasha-solomon, as discussed on a call - maybe we can just have a small section, that states: use can add to custom highlighted fields any filed returned by aggr/non-aggr queries(ref to them?), including newly created fields

Reply:
@vitaliidm would you mind checkign [c64371a](https://github.com/elastic/security-docs/pull/5182/commits/c64371a98f9a0d1ad2dfd723223b7699796b2a5c) and letting me know if it fits what you're suggesting?